tee-worker/litentry/pallets/identity-management/src/lib.rs

```diff
@@ -317,6 +326,28 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
+		fn insert_identity_with_limit(
```

Comment:
we should use this fn too when inserting the prime identity => so that the `AccountIdentityContextCount` gets incremented. Ideally we could have a test case for it :)

Reply:
I'll add relevant storage assertions in tests